src/utils/lock_wrapper.h

```diff
@@ -248,19 +242,15 @@ class lock_spin_recursive : public lock_spin
 		return ret;
 	};
 	inline int unlock() {
-#ifdef DEFINED_NO_THREAD_LOCK
-		return 0;
-#endif // DEFINED_NO_THREAD_LOCK
+		DEFINED_NO_THREAD_LOCK_RETURN
 		if (--m_lock_count == 0) {
 			m_owner = m_invalid_owner;
 			return lock_spin::unlock();
 		}
 		return 0;
 	};
 	inline int is_locked_by_me() {
-#ifdef DEFINED_NO_THREAD_LOCK
-		return 1;
-#endif // DEFINED_NO_THREAD_LOCK
+		DEFINED_NO_THREAD_LOCK_RETURN
```

Comment:
@liranoz12 @OphirMunk in original code this function returns 1 but in new return 0. Is it expected?